src/devices/Ht16k33/Ht16k33.cs

```diff
@@ -0,0 +1,162 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.I2c;
+using Iot.Device.Ht16k33.Enums;
+
+namespace Iot.Device.Ht16k33
+{
+    /// <summary>
+    /// HT16K33 LED driven display driver
+    /// </summary>
+    /// <remarks>
+    /// Datasheet: https://www.adafruit.com/datasheets/ht16K33v110.pdf
+    /// Sources:
+    /// https://github.com/adafruit/Adafruit_LED_Backpack/blob/master/Adafruit_LEDBackpack.cpp
+    /// https://github.com/sobek1985/Adafruit_LEDBackpack/blob/master/Adafruit_LEDBackpack/AlphaNumericFourCharacters.cs
+    /// </remarks>
+    public abstract class Ht16k33 : IDisposable
+    {
+        #region Constants
+        /// <summary>
+        /// HT16K33 default I2C address
+        /// </summary>
+        public const int DefaultI2cAddress = 0x70;
+        #endregion
+
+        #region Protected members
+        /// <summary>
+        /// I2C device interface
+        /// </summary>
+        protected I2cDevice _i2cDevice;
+
+        /// <summary>
+        /// Initial brightness level
+        /// </summary>
+        protected byte brightness;
```

Comment:
this should be one of following:
- private field with prefixed with `_`
- protected property with regular capital case

Reply:
similarly for other fields

Reply:
Done.